app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala

```diff
@@ -89,6 +89,7 @@ class AuditService @Inject()(auditConnector: DefaultAuditConnector) {
     sendExplicitAuditEvent(
       auditEventType = RulingCancelled,
       auditPayload = statusChangeAuditPayload(oldCase, updatedCase, operator)
+        + ("cancelReason" -> cancelReason(updatedCase))
```

Comment:
can be written in the line above (no need of a new line)

Reply:
I thought I've seen warnings about excessively long lines.  I prefer to keep it on the screen (unless we feel strongly about this)